src/components/PeopleTable.tsx

```diff
@@ -1,682 +1,73 @@
-export const PeopleTable = () => {
+import classNames from 'classnames';
+
+import { Person } from '../types';
+import { OrderOption, SortOptions } from '../utils/data';
+import { PersonRow } from './PersonRow';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  people: Person[];
+  sort: SortOptions;
+  order: OrderOption;
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  sort,
+  order,
+}) => {
+  const sortParams = (option: SortOptions) => {
+    if (!sort || sort !== option) {
+      return { sort: option, order: null };
+    }
+
+    if (sort === option && !order) {
+      return { sort: option, order: 'desc' };
+    }
+
+    return { sort: null, order: null };
+  };
+
   return (
     <table
       data-cy="peopleTable"
       className="table is-striped is-hoverable is-narrow is-fullwidth"
     >
       <thead>
         <tr>
-          <th>
-            <span className="is-flex is-flex-wrap-nowrap">
-              Name
-              <a href="#/people?sort=name">
-                <span className="icon">
-                  <i className="fas fa-sort" />
-                </span>
-              </a>
-            </span>
-          </th>
-
-          <th>
-            <span className="is-flex is-flex-wrap-nowrap">
-              Sex
-              <a href="#/people?sort=sex">
-                <span className="icon">
-                  <i className="fas fa-sort" />
-                </span>
-              </a>
-            </span>
-          </th>
-
-          <th>
-            <span className="is-flex is-flex-wrap-nowrap">
-              Born
-              <a href="#/people?sort=born&amp;order=desc">
-                <span className="icon">
-                  <i className="fas fa-sort-up" />
-                </span>
-              </a>
-            </span>
-          </th>
-
-          <th>
-            <span className="is-flex is-flex-wrap-nowrap">
-              Died
-              <a href="#/people?sort=died">
-                <span className="icon">
-                  <i className="fas fa-sort" />
+          {Object.values(SortOptions).map(option => {
+            return (
+              <th key={option}>
+                <span className="is-flex is-flex-wrap-nowrap">
+                  {option.charAt(0).toUpperCase() + option.slice(1)}
```

Comment:
To follow best practices, you should create a function to capitalize the first letter of a string and use it here.